MIDAS/src/systems.cpp

```diff
@@ -34,6 +34,15 @@ DECLARE_THREAD(low_g, RocketSystems* arg) {
     vTaskDelete(NULL);
 }
 
+DECLARE_THREAD(gyroscope, RocketSystems* arg) {
+    while (true) {
+        THREAD_SLEEP(16);
```

Comment:
one last thing sorry try to match the max sampling rate of the sensor with the thread sleep

Reply:
punks